.github/workflows/test.yml

```
@@ -25,7 +25,7 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         run: npm ci
 
-      - name: Run linting
+      - name: Run lintingg
```

Minor spelling mistake

```suggestion
      - name: Run linting
```

---

I know, I added it to create a new PR. I thought it would make the tests pass in the other one. This one isnt really needed anymore since tests failed anyway. I;ll close it